packages/library-export-default-webpack-plugin/package.json

```diff
@@ -24,6 +24,7 @@
 	"files": [
 		"index.js"
 	],
+	"main": "index.js",
```

Comment:
Technically redundant (it's the default), but I guess to be explicit?

Reply:
Yes, as discussed in https://github.com/WordPress/gutenberg/pull/15879#discussion_r288714531.